src/modules/commander/CMakeLists.txt

```diff
@@ -30,16 +30,12 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 ############################################################################
-set(MODULE_CFLAGS -Os)
-if(${OS} STREQUAL "nuttx")
-	list(APPEND MODULE_CFLAGS -Wframe-larger-than=2450)
-endif()
 px4_add_module(
 	MODULE modules__commander
 	MAIN commander
-	STACK 4096
+	STACK_MAIN 4096
+	STACK_MAX 2450
```

Comment:
@LorenzMeier This doesn't make any sense to me, why are we warning if the stack is larger than 2450 but the main stack is set to 4096?